src/prefect/agent/agent.py

```diff
@@ -181,12 +183,9 @@ def on_shutdown(self) -> None:
         as a hook for child classes to optionally implement.
         """
 
-    def agent_connect(self) -> str:
+    def agent_connect(self) -> None:
```

Comment:
Now that the tenant id query is removed this should perform some basic query to verify that it is connected / authenticated. Otherwise in this current implementation it's just for show

Reply:
Are you OK if we punt that to a different PR @joshmeek ?